File: app/src/main/java/hottopic/mit/co/nz/cleaningservice/entities/UserInfo.java
package hottopic.mit.co.nz.cleaningservice.entities;

import java.io.Serializable;

public class UserInfo implements Serializable{
    private String userId;
    private String userName;
    private String phoneNumber;
    private String email;
    private Address address;
    private UserRole userRole;
    private float balance;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public Address getAddress() {
        return address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }

    public UserRole getUserRole() {
        return userRole;
    }

    public void setUserRole(UserRole userRole) {
        this.userRole = userRole;
    }

    public float getBalance() {
        return balance;
    }

    public void setBalance(float balance) {
        this.balance = balance;
    }
}

File: app/src/main/java/hottopic/mit/co/nz/cleaningservice/entities/users/UserInfo.kt
package hottopic.mit.co.nz.cleaningservice.entities.users

import java.io.Serializable

data class UserInfo(var username: String?, var password: String?, var phone: String?,
               var email: String?, var city: String?, var suburb: String?, var street: String?,
               var role_id: Int, var balance: Float) : Serializable {
    var id: Int = 0
    var role_name: String? = null

    val address: String
        get() = "$street, $suburb, $city"

    fun formatBalance(): String {
        return "${this.balance}"
    }
}
